scripts/postdeploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd app
+
+# Run Django migrations.
+python manage.py migrate
+# Seed database with dummy users
+python manage.py seedusers
```

Comment:
`pr-predestroy` is documented, so my guess is that there is an undocumented release phase hook `pr-postdeploy`. From what I understand, yes, all env are carried over to every app that uses it.